src/main/java/com/ibm/watson/developer_cloud/conversation/v1_experimental/model/MessageResponse.java

```diff
@@ -93,6 +94,24 @@ public void setLocation(Integer... location) {
     public void setValue(String value) {
       this.value = value;
     }
+
+    @Override public boolean equals(Object obj) {
```

Comment:
The object extends `GenericModel` so it has `equals` implemented and uses the json representation for that.
Why do you want to redefine that ?